File: mycollab-services/src/main/java/com/esofthead/mycollab/configuration/DatabaseConfiguration.java
package com.esofthead.mycollab.configuration;

public class DatabaseConfiguration {

	private String driverClass;

	private String dbUrl;

	private String user;

	private String password;

	DatabaseConfiguration(String driverClass, String dbUrl, String user,
			String password) {
		this.user = user;
		this.driverClass = driverClass;
		this.dbUrl = dbUrl;
		this.password = password;
	}

	public String getDriverClass() {
		return driverClass;
	}

	public String getDbUrl() {
		return dbUrl;
	}

	public String getUser() {
		return user;
	}

	public String getPassword() {
		return password;
	}
}

File: mycollab-config/src/main/scala/com/esofthead/mycollab/configuration/DatabaseConfiguration.scala
package com.esofthead.mycollab.configuration

import scala.beans.BeanProperty

/**
 * @author MyCollab Ltd
 * @since 5.1.0
 */
class DatabaseConfiguration(@BeanProperty val driverClass:String, @BeanProperty val dbUrl:String,
                            @BeanProperty val user:String, @BeanProperty val password:String) {
    
}
